modal_proto/api.proto

```diff
@@ -20,6 +20,9 @@ enum AppState {
 
   // Stopped
   APP_STATE_STOPPED = 5;
+
+  // App is created and in process of deployment.
+  APP_STATE_DEPLOYING = 6;
```

Comment:
just so I understand it, this is only the first time you deploy an _new_ app right? when you're deploying to an existing app, it won't go to this state?

Reply:
Yeh that's the intention. Just need it to differentiate a new deploying app. Deployed apps are existing, and already in `DEPLOYED` state.